app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other) -> int:
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other) -> int:
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+        self.km += other.km
+        return self
+
+    def __mul__(self, other) -> int:
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+        return Distance(self.km * other.km)
+
+    def __truediv__(self, other) -> float:
+        if isinstance(other, (int, float)):
+            return Distance(round(self.km / other, 2))
+        return Distance(round(self.km / other.km, 2))
+
+    def __lt__(self, other) -> bool:
+        if isinstance(other, (int, float)):
+            return True if self.km < other else False
+        return True if self.km < other.km else False
+
+    def __gt__(self, other) -> bool:
+        if isinstance(other, (int, float)):
+            return True if self.km > other else False
+        return True if self.km > other.km else False
+
+    def __eq__(self, other) -> bool:
+        if isinstance(other, (int, float)):
+            return True if self.km == other else False
+        return True if self.km == other.km else False
+
+    def __le__(self, other) -> bool:
+        if isinstance(other, (int, float)):
+            return True if self.km <= other else False
```

Comment:
```suggestion
            return self.km <= other
```
Such construction can be simplified since `self.km <= other` statement already returns a boolean value

Reply:
please, fix all of them